src/Glimpse.Server.Web/Resources/History.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Glimpse.Server.Web.Extensions;
+using Microsoft.AspNet.Http;
+using Microsoft.Net.Http.Headers;
+
+namespace Glimpse.Server.Web
+{
+    public class History : IResource
```

Comment:
So far the pattern I have been following for resources is calling them `HistoryResource`. Doesn't mean we need to, but this one would be against the grain atm.